webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard/ChangeHostCVModal.js

```diff
@@ -174,13 +175,22 @@ const ChangeHostCVModal = ({
           <a href={`/change_host_content_source?host_id=${hostId}`}>{__('change the host\'s content source.')}</a>
         </Alert>
       }
+      {multiEnv &&
+        <Alert
+          variant="warning"
+          ouiaId="multi-env-alert"
+          isInline
+          title={__('This host is associated with multiple content view environments. If you assign a lifecycle environment or content view here, the host will be removed from the other environments.')}
```

Comment:
```suggestion
          title={__('This host is associated with multiple content view environments. If you assign a lifecycle environment and content view here, the host will be removed from the other environments.')}
```
Should this be `and`?

Reply:
Updated :+1: